Code Jam 8 - Initial PR for Announcement

janine9vn opened the pull request:
This Pull Request is just the basics required for the Code Jam announcement. It does the following things:
1. Add in basic assets for the event
2. Create the CJ8 pages
3. Update the main events page and the main code jam page

There will be a second PR made this Sunday that will have more of the details and include the Qualifier release.

janine9vn committed:
Code Jam 8 Event Images
Added images for the event, including the website banner
and images for the different approved frameworks.

janine9vn committed:
Update list of upcoming and previous code jams
Updating the list for upcoming and previous code jams.
This includes the overall "Upcoming Events" on the main events page.

I will do better in the future to update this as we have upcoming events

janine9vn committed:
Update main event page to draw more attention
Making it more obvious that we have an ongoing code jam
that is available to join soon.

janine9vn committed:
Create main Code Jam 8 event page
Created the main code jam event page.
It has barebones information with another more substantial
update to be PRed this upcoming Sunday.

janine9vn committed:
Create frameworks page for CJ8
Created a page listing the approved frameworks for CJ8.

janine9vn committed:
Create rules page for CJ8

janine9vn committed:
Update sidebar link for CJ8
This changes the look of the sidebar.
I've decided to put the Important Links in a panel item.
This makes it more obvious and draws more attention to it.

Additionally, I've put it above the sponsors for the above reasons.

jb3 commented on pydis_site/templates/events/pages/code-jams/8/_index.html:
```diff
@@ -0,0 +1,43 @@
+
+    <h3 id="important-dates"><a href="#important-dates">Important Dates</a></h3>
+    <ul>
+        <li>Monday, June 14 - Form to submit theme suggestions opens</li>
```
```suggestion
        <li>Monday, June 15 - Form to submit theme suggestions opens</li>
```

janine9vn committed:
Update release date for theme suggestions form

Co-authored-by: Joe Banks <joseph@josephbanks.me>

Shivansh-007 commented on pydis_site/templates/events/pages/code-jams/8/_index.html:
```diff
@@ -0,0 +1,43 @@
+
+    <h3 id="how-to-join"><a href="#how-to-join">How to Join</a></h3>
+    <p>The Qualifier isn't released yet, but to receive the most up-to-date information and to get notified 
+        when the Qualifier released you can join the server: <a href="https://discord.gg/python">discord.gg/python</a>.</p>
```
```suggestion
        when the Qualifier is released you can join the server: <a href="https://discord.gg/python">discord.gg/python</a>.</p>
```

jb3 reviewed (approved):
Other than linting failure, all looks good, approving.

Shivansh-007 commented on pydis_site/templates/events/pages/code-jams/8/rules.html:
```diff
@@ -0,0 +1,70 @@
+    <li>
+        <p>
+            Your team, once the coding portion of the code jam is complete, must create a video presentation that showcases and explains your final product.
+            This must be in a video format and must be uploaded somewhere for the judges to view (i.e. unlisted youtube video, vimeo, etc.)
```
```suggestion
            This must be in a video format and must be uploaded somewhere for the judges to view (i.e. unlisted YouTube video, Vimeo, etc.)
```

jacobmonck reviewed (approved):
Looks good to me

Shivansh-007 reviewed (approved):
Wait

janine9vn committed:
Removed rogue trailing whitespaces

janine9vn committed:
Merge remote-tracking branch 'origin/cj8_announcement' into summer-code-jam

janine9vn committed:
Fix accidentally a word

Co-authored-by: Shivansh-007 <shivansh-007@outlook.com>

whispersofthedawn reviewed (approved):
LGTM

janine9vn committed:
Remove trailing whitespace and fix capitalization
Trailing whitespace got merged with a commit made through
the Github UI editor. Never again shall I trust it!

Also capitalized Vimeo and Youtube.
